### PR TITLE
OSDOCS-10915: adds 4.14.32 relnotes Microshift

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.14

Issue:
[OSDOCS-10915](https://issues.redhat.com/browse/OSDOCS-10915)

Link to docs preview:
[4.14.32 async relnote](https://78535--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-14-release-notes.html#microshift-4-14-32-dp)

QE review:
- NA stock text

### CODE REVIEW

**Comment by aravipra** [comment]
Do not merge. Only peer review is needed.

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 1]
```diff
@@ -566,4 +566,13 @@ Issued: 2024-06-26
 
 {product-title} release 4.14.31 is now available. The list of enhancements and bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:4012[RHBA-2024:4012] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:4010[RHSA-2024:4010] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-14-32-dp"]
```
Comment (skopacz1) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:571]: Just a heads up that all anchors (except for the anchor of an assembly's title) require {context} at the end of them, per the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings). You don't have to fix existing instances in this file if you don't want to, but please add it to this ID:

```suggestion
[id="microshift-4-14-32-dp_{context}"]
```

**Review by skopacz1 — COMMENTED** [review]
One small comment, otherwise LGTM

File: microshift_release_notes/microshift-4-14-release-notes.adoc (hunk 1 above)
Comment (aravipra) [reply microshift_release_notes/microshift-4-14-release-notes.adoc]: done

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**